pkgs/development/python-modules/pydiscourse/default.nix

```
@@ -0,0 +1,34 @@
+{
+  buildPythonPackage,
+  fetchFromGitHub ,
+  lib,
+  pythonAtLeast,
+  requests,
+}:
+
+buildPythonPackage rec {
+  pname = "pydiscourse";
+  version = "1.4.0";
+  format = "setuptools";
+
+  # the tox.ini file sets the tests up for Python 3.[7-10] only
+  patches = [ ./0001-Enable-the-tests-for-Python-3.11.patch ];
+
+  src = fetchFromGitHub {
+    owner = "pydiscourse";
+    repo = pname;
+    rev = "refs/tags/v${version}";
+    hash = "sha256-peDkXRcD/ieWYWXqv8hPxTSNRXBHcb/3sj/JJSF2RYg=";
+  };
+
+  propagatedBuildInputs = [ requests ];
+
+  pythonImportsCheck = [ "pydiscourse" ];
+
+  meta = with lib; {
+    homepage = "https://github.com/pydiscourse/pydiscourse";
+    description = "A Python library for working with Discourse.";
```

```suggestion
    description = "A Python library for working with Discourse";
```

No period at the end please.